lib/logstash/inputs/s3sqs.rb

```diff
@@ -6,6 +6,9 @@
 require "logstash/plugin_mixins/aws_config"
 require "logstash/errors"
 
+# https://aws.amazon.com/blogs/developer/threading-with-the-aws-sdk-for-ruby/.
+# Aws.eager_autoload!
```

Comment:
is this necessary?

Reply:
I think so - I saw some weird dependency issues during testing, which didn't happen when I reinstated this line.